### ISSUE TITLE
Provide an enVironment Manager tool

### ISSUE THREAD

**Issue by danielpetisme** [eclipse/golo-lang#37]
As POC I've made the [govm](https://github.com/danielpetisme/govm) (stands fo Golo enVironment Manager). The tool is a fork + sed 's/gvm/govm/g' of the [gvm](https://github.com/gvmtool/gvm) tool.

I think an enVironment Manager would be a great killer feature for golo adoption, so it should be managed though the official Github account.

Now, regarding the implementation
govm, gvm client side is obsviously do in shell (compliant bash & cygwin). For server side Vert.x + mongoDB (currenlty hosted on heroku). I ask to the gvm manager if it would be possible to have an environment manager techno agnostic (xvm). Adding a custom wrapper on top of it.

@jponge proposed [jenv](https://github.com/hikage/jenv) (inspired by rbenv). The installation seems a bit harder. I dunno how the backend is managed.

So basically we have to choose:
- the hosting behavior (heroku, cloudbees, custom?)
- a basic tool to reuse (gvm/jenv)
